SalesforceCommerceCloud/pwa-kit
# [Application Extension] Use the configured extensions only (@W-17151247@)

To simplify our implementation, we will no longer have both concepts of _installed_ and _configured_ extensions in our code. Simply use the configured extensions only.

PR of previous iteration: #2112 


## How to test drive the PR

1. `npm ci` at the monorepo root
2. `npm start` at the typescript-minimal project
3. Verify that the project renders with a blue border around the app. That's the result of the sample extension enabled.
4. Now edit the package.json to disable the sample extension
5. Wait for the dev server to rebuild. No need to restart the server because the configured extensions are also parsed during runtime.
6. Reload your browser and verify that the homepage no longer has the blue border.
